sqrl-calcite/src/main/java/com/datasqrl/plan/validate/ScriptPlanner.java

```diff
@@ -256,6 +250,9 @@ public Void visit(SqrlExportDefinition node, Void context) {
               exportRelNode.getRowType(), List.of(), LogFactory.Timestamp.NONE);
           tableSink = sinkLog.getSink();
         }
+      } else if (connector == SystemBuiltInConnectors.LOG_ENGINE) {
```

Comment:
So if I do a `EXPORT MyTable TO log.Table`, it only works if I already have the log defined?

Reply:
Yes, I believed this approach was logical. For instance, when Vertx is enabled, this log table will exist. I thought maintaining this order would be beneficial.